src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -46,8 +46,19 @@ export const useFormSend = () => {
         } else if (adapterType === ChainTypes.Bitcoin) {
           const fees = estimatedFees[feeType] as chainAdapters.FeeData<ChainTypes.Bitcoin>
 
-          const accountType = accountIdToAccountType(data.accountId)
-          const utxoParams = utxoAccountParams(data.asset, accountType, 0)
+          const { accountType, utxoParams } = accountIdToUtxoParams(data.asset, data.accountId, 0)
+
+          if (!accountType) {
+            throw new Error(
+              `useFormSend: could not get accountType from accountId: ${data.accountId}`
+            )
+          }
+
+          if (!utxoParams) {
```

Comment:
isEmpty?

Reply:
good question - but `accountIdToUtxoParams` will return an empty object, which when destructured `utxoParams` is undefined